src/pages/Compare.tsx

```diff
@@ -1,17 +1,33 @@
 import { useEffect } from "react";
-import { Navigate, useSearchParams } from "react-router-dom";
+import { Navigate, useLocation, useSearchParams } from "react-router-dom";
 import { Grid, GridContainer } from "@trussworks/react-uswds";
+import { Marker } from "react-leaflet";
 
+import BackButton from "../components/BackButton";
+import ResultsMap from "../components/Search/ResultsMap";
+import ShareButton from "../components/ShareButton";
+import { getMapMarker, getResultBounds } from "../utils";
 import CARE_PROVIDER_DATA from "../data/ladders_data.json";
 import { CareProvider } from "../types";
 import { logPageView } from "../analytics";
+import { useTranslation } from "react-i18next";
+import CompareDetail from "../components/ResultDetail/CompareDetail";
 
 export default function Compare() {
   useEffect(() => {
     logPageView();
   }, []);
 
   const [params, _setParams] = useSearchParams();
+  const location = useLocation();
+  const { t } = useTranslation();
+
+  // TODO: make this page zip-aware
```

Comment:
what does 'zip aware' mean?

Reply:
have zip from the search page as a url param and display distance from zip in the location summary